sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 1.2.0-preview.2 (Unreleased)
-
+- Updating `DefaultAzureCredential` to enable authenticating through Visual Studio
```

Comment:
Updated ?

Reply:
I honestly don't know what the tense of this should be. Looking back through the changelog it seems to trend to the past tense, but goes back and forth, and a bunch of entries mix past tense and present tense. Do you know of any guidelines which prefer one or the other? In absence of that, I suppose I'll change this to past tense and try to stick to this convention.